### DISCUSSION TITLE
How do I log in?

### DISCUSSION THREAD

**Discussion by nmalancea** [gladly-team/next-firebase-auth#447] [Q&A] [answered]
When I call `firebase.auth().signInWithEmailAndPassword(email, password)` I get "function does not exist" (because it is only [meant](https://stackoverflow.com/a/48706998/1678614) to be used in the browser, but not server-side ?).

What is the right way to log in with `next-firebase-auth`?
Any chance we could get an example that does not rely on `firebase-ui`?

**Comment by kmjennison** [comment, marked as answer]
Firebase login is the same as it would be without using `next-firebase-auth`. Please be sure you're using the Firebase JS SDK (not the Firebase admin app) and calling it only on the client side.

If you're still having problems, please share a reproduction or more info here so that others might be able to help.

**Reply by nmalancea** [reply]
Thank you!
My issue is that I was using `useAuthUser` without wrapping the component in `withAuthUser`.
